### ISSUE TITLE
Example in README broken on recommended release

### ISSUE THREAD

**Issue by Firionus** [redox-os/orbtk#248]
**Describe the bug**
Minimal example in README.md is not compatible with release recommended in README.md. 

**To Reproduce**
1. `cargo new orbtk_minimal`
2. add `orbtk = "0.3.1-alpha1"` to dependencies in Cargo.toml as recommended in README.md
3. Copy minimal example from README.md into main.rs, replacing existing Hello-World-code
4. `cargo run`
5. Get compiler error
```
error[E0599]: no function or associated item named `new` found for struct `orbtk_widgets::window::Window` in the current scope
 --> src\main.rs:6:21
  |
6 |             Window::new()
  |                     ^^^ function or associated item not found in `orbtk_widgets::window::Window`

error[E0599]: no function or associated item named `new` found for struct `orbtk_widgets::text_block::TextBlock` in the current scope
  --> src\main.rs:10:35
   |
10 |                 .child(TextBlock::new().text("OrbTk").build(ctx))
   |                                   ^^^ function or associated item not found in `orbtk_widgets::text_block::TextBlock`

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0599`.
error: could not compile `orbtk_minimal`.
```
**Expected behavior**
Example in README.md should compile with release recommended in README.md

**Desktop (please complete the following information):**
 - OS: Windows 10 Home 1903, Build 18362.900

**Additional context**
Replacing `orbtk = "0.3.1-alpha1"` with `orbtk = { git = "https://github.com/redox-os/orbtk.git", branch = "develop" }` in Cargo.toml fixes compilation and a Window with the text "OrbTk" opens, as expected.

**Comment by FloVanGH** [comment]
Thank you for the report. I will fix it.

**Comment by FloVanGH** [comment]
@Firionus I see. There are some API changes that not included in `0.3.1-alpha1`. If you want to run the example from the Readme you have to use `orbtk = { git = "https://github.com/redox-os/orbtk, branch = "develop" }` until `0.3.1-alpha3` is released.
